### PR TITLE
PIN-3325 [BE] Aggiornare la configurazione dell'analisi di rischio dei privati

### PR DESCRIPTION
**Pull request by nttdata-rtorsoli**

### CODE REVIEW

File: src/main/resources/riskAnalysisTemplate/forms/PRIVATE/2.0.json
Comment (galales) [inline src/main/resources/riskAnalysisTemplate/forms/PRIVATE/2.0.json:1]: the given risk analysis is the 1.0 for PRIVATE

File: src/test/scala/it/pagopa/interop/purposeprocess/RiskAnalysisValidationSpec.scala [hunk 1]
```diff
@@ -12,103 +12,147 @@ import it.pagopa.interop.tenantmanagement.client.model.TenantKind
...
 import org.scalatest.Assertion
 import org.scalatest.matchers.should.Matchers._
 import org.scalatest.wordspec.AnyWordSpecLike
+import it.pagopa.interop.purposeprocess.model.RiskAnalysisForm
 
 class RiskAnalysisValidationSpec extends AnyWordSpecLike {
   implicit val eqError: Eq[RiskAnalysisValidationError] = Eq.fromUniversalEquals
 
   "Risk Analysis Validation" should {
-    "succeed on correct form 1.0" in {
```
Comment (galales) [inline src/test/scala/it/pagopa/interop/purposeprocess/RiskAnalysisValidationSpec.scala:24]: ⚠️ this was the test for the version 1.0 for PAs, it should not be migrated to 2.0.
The test for the PAs 2.0 should be a different test

File: src/test/resources/riskAnalysisTemplate/forms/PRIVATE/2.0.json
Comment (galales) [inline src/test/resources/riskAnalysisTemplate/forms/PRIVATE/2.0.json:1]: same for other PR, this should not be duplicated in test package

**Review by galales — CHANGES_REQUESTED** [review]

File: src/main/resources/riskAnalysisTemplate/forms/PRIVATE/2.0.json
Comment (nttdata-rtorsoli) [reply src/main/resources/riskAnalysisTemplate/forms/PRIVATE/2.0.json:1]: ? with a file json named 2.0 with inside "version": "2.0"?

**Review by galales — APPROVED** [review]

**Review by beetlecrunch — APPROVED** [review]

**Review by galales — APPROVED** [review]

**Pull request merged**